lib/controller.js

```diff
@@ -113,7 +114,8 @@ Tessel.get = function(opts) {
     var seekerOpts = {
       timeout: (opts.timeout || 2) * 1000,
       usb: opts.usb,
-      lan: opts.lan
+      lan: opts.lan,
+      authorized: opts.authorized || true
```

Comment:
If `opts.authorized` is explicitly set to `false`, this expression will evaluate to `true`, ie. `false || 1; // 1`